blueprinter.gemspec

```diff
@@ -25,4 +25,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rspec", "~> 3.7"
   s.add_development_dependency "sqlite3"
   s.add_development_dependency "yard", "~> 0.9.11"
+  s.add_development_dependency "factory_bot"
```

Comment:
would you move this up to retain ABC ordering?

Reply:
will do, I wasn't aware of the ordering!